database_admin/migrations/088_bigint_repo.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE repo ALTER COLUMN id TYPE BIGINT;
+
+ALTER TABLE system_repo ALTER COLUMN repo_id TYPE BIGINT,
+                        ALTER COLUMN system_id TYPE BIGINT,
+                        ALTER COLUMN rh_account_id TYPE BIGINT;
```

Comment:
Wow, you did this so smoothly, do you want to migrate other tables since you're on it? :trollface: